### PR TITLE
Portfolio popup window

### PR DESCRIPTION
**Pull request by principles31**
**Portfolio: details popup window**

-The goal of this project is to implement a popup window that includes the project details on both desktop and mobile. 
- The following features have been implemented in the project.
> When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
> When the user clicks (or taps) the close (X) button, the popup disappears.
> The required changes requested by the previous reviewer have been implemented.

> popup desktop has been added to the project.
> popup mobile has been added to the project.
> Template 5 was used.
. Below is a screenshot of the project.
**- Mobile popup**
- popup_!
**- Desktop popup**
popup_2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,84 +2,181 @@
...
-            </section>
+            </div>
+
+            <div class="mobile_image"></div>
+        </section>
 
+        
             <div class="grid_item_work grid_2_item">
                 <!-- work section  -->
                 <section id="portfolio" class="works_section">
```
Comment (topeogunleye) [inline index.html:175]: - [x] [REQUIRED] As seen in the project requirement. I kindly suggest that you create a project's `array of objects` to store your project's information and create the projects dynamically in JS. Modal popup data will also be populated with the data from the projects array.

### EXAMPLE
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

Each array object should contain at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @principles31,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Github flow is used :heavy_check_mark:.
> * You documented your work professionally :heavy_check_mark: .
> 
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (principles31) [reply index.html:175]: @topeogunleye Thank you for the feedback. i will implement the above issues.

File: index.html [hunk 2]
```diff
@@ -2,265 +2,400 @@
...
+            <div class="project_grid_row">
+                <h1 class="project_header text_color text_body_font m_header">Project name goes here</h1>
+            </div>
+            <div class="project_grid_row grid_img_2">
+                <ul class="project_category list_category text_body_font">
+                    <li>Html</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+            </div>
```
Comment (smunozmo) [inline index.html]: - [x] Good work creating the projects array! However, as the previous reviewer said, the goal here is to load the projects' data from the `projectlist1` array, right now the info displayed on the modal window is hardcoded:

Data displayed:
Captura de pantalla 2022-04-16 a la(s) 3 44 11 p  m

Data that should be displayed:
Captura de pantalla 2022-04-16 a la(s) 3 39 01 p  m

- Please remove the hardcoded data as the previous CR said and populate it from the array, this way your site will be more dynamic.

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @principles31, 👋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._




Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by principles31** [comment]
**Portfolio: validate contact form**
-The goal of this project is to implement a client-side validation of the data in the contact form before submission.
-The following features have been implemented in the project
**.**You need to implement a simple validation:

**-**The content of the email field has to be in lower case.
**-**You should implement the following interactions:

-When the user submits the form, you check if the email is in lower case.
-If the validation is OK, the form is sent.
-If the validation is not OK, you show an error message to the user near the submit button informing them of the error, and the form is not sent.
A validation Message was added to this pull request.
-A screenshot from template 5 has been added.
error_1
.

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @principles31  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by smunozmo — APPROVED** [review]
Hi @principles31, 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 👏 👏 👏 




Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (topeogunleye) [reply index.html:175]: You're welcome

**Review by Meltrust — COMMENTED** [review]
Hi @principles31,



## Wow, you did it 🎉 
Brilliant



### To highlight:

- The project works✔️ 
- Nice readme✔️ 
- Good code organization✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — COMMENTED** [review]
Hi @principles31, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by principles31** [comment]
@keroloussamy Thank you keep the good work

**Pull request merged**